dnf/cli/option_parser.py

```diff
@@ -258,13 +258,22 @@ def _add_general_options(self):
                                                       " for all questions"))
         general_grp.add_argument("--enablerepo", action=self._RepoCallback,
                                  dest='repos_ed', default=[], metavar='[repo]',
-                                 help=_("Enable additional repositories. List option. "
-                                        "Supports globs, can be specified multiple times."))
+                                 help=_("Temporarily enable repositories for the purpose"
+                                        "of the current dnf command. Accepts an id, a"
+                                        "comma-separated list of ids, or a glob of ids."
+                                        "This option can be specified multiple times."
+                                        "To permanently enable new repositories, use"
+                                        "`dnf config-manager --enable <repo id>`"))
```

Comment:
Unfortunately, `dnf config-manager` is actually not yet part of the dnf core, so we can't refer to it here, as it's entirely possible that isn't present at all.